doc/sphinx/source/community/broken_recipe_policy.rst

```diff
@@ -0,0 +1,17 @@
+Broken recipe policy
+====================
+
+Recipes might stop working for different reasons. Among those are, for instance, withdrawal of datasets
```

Comment:
It would be good to make it a bit more explicit here that a recipe that contains data that is no longer publicly available is considered broken and needs to be fixed. This appears not to be clear at the moment: https://github.com/ESMValGroup/ESMValTool/issues/2684#issuecomment-1169620612.

Reply:
Maybe like this?

`[...] withdrawal of datasets used by the recipe (i.e. the recipe contains data that are no longer publicly available) [...]`

Reply:
Sounds good! And what do we think about datasets that have been retracted, but are still available for download? Did we make a decision on those at the last workshop?